src/acl.c

```diff
@@ -1439,7 +1439,7 @@ int ACLCheckUserCredentials(robj *username, robj *password) {
 /* If `err` is provided, this is added as an error reply to the client.
  * Otherwise, the standard Auth error is added as a reply. */
 void addAuthErrReply(client *c, robj *err) {
-    if (clientHasPendingReplies(c)) return;
+    if (c->flag.buffered_reply) return;
```

Comment:
Can we just improve the function documentation by explaining this logic. it was also missing before

Reply:
do you like the way i do it now? Or can you suggest a text so that i can copy it